docs/site/submitting_a_pr.md

```diff
@@ -0,0 +1,527 @@
+[Conventional Commits](https://conventionalcommits.org/).
+
+Please read the
+[Commit Message Format](https://loopback.io/doc/en/contrib/code-contrib-lb4.html#commit-message-guidelines)
+guidelines to correctly format your commit messages.
+
+To change an existing commit please refer to
+[Changing a commit message](https://help.github.com/articles/changing-a-commit-message/).
+
+{% include note.html content="Run <b>npm install</b> inside <b>loopback-next</b> after <b>git clone</b>. This will automatically set up git <b>commit-msg</b> hooks to check the conventions and block invalid messages.
+" %}
+
+#### 2. Before pushing to remote repository
+
+Before pushing to the remote repository, ensure your files are free of
+formatting, syntax, and logical/execution errors by running:
+
+```
+npm run lint:fix && npm test
+```
+
+{% include note.html content="<b>npm run lint:fix</b> might reformat the source code and fix style issues. Be sure to add such changes to your commit." %}
+
+#### 3. After creating PR, sign the CLA and fill out checklist
+
+After creating the pull request, make sure the Contributor License Agreement
+(CLA) has been signed for the `loopback-next` repository.
+
+The pull request has a checklist that must be filled out appropriately.
+
+![submit_pr_template_1.png](./imgs/submit_pr_template_1.png)
+
+Of particular importance are the
+`'Read and sign the CLA (Contributor License Agreement)'` link and the
+`'npm test passes on your machine'` checkbox.
+
+Click on the
+[Read and sign the CLA (Contributor License Agreement)](https://cla.strongloop.com/agreements/strongloop/loopback-next)
+link, and sign it. This is done once per repository.
+
+#### 4. Check CI status
+
+Ensure that the continuous integration (CI) jobs associated with your pull
+request complete successfully.
+
+It might take some time for CI jobs to be scheduled and completed.
+
+![submit_pr_final_commit_ci_jobs_pass_1.png](./imgs/submit_pr_final_commit_ci_jobs_pass_1.png)
+
+If a build fails, click on its `Details` link for more information.
+
+For example, clicking on the `Details` link for the `Travis CI build` job takes
+us to the Travis CI build page.
+
+![submit_pr_travis_ci_build_page_1.png](./imgs/submit_pr_travis_ci_build_page_1.png)
+
+Here you can investigate the status of the job, its sub-jobs, or restart them.
+
+#### 5. PR Review Process
+
+The review process is iterative, so it is normal that `multiple commits` may be
+required until the pull request is finally accepted.
+
+{% include tip.html content="After a pull request is created, any additional commits that you push to your remote feature branch automatically get added to the pull request. There is no need to close the initial pull request and create a new one." %}
+
+Reactively [rebase](https://help.github.com/articles/about-git-rebase/) your
+forked repository against the upstream master branch to keep them in sync; if
+needed.
+
+#### Before your PR is merged by a maintainer
```

Comment:
Missing the item number such as `#### 6. ...`

Reply:
good catch. How did I miss that? oh well.